### PR TITLE
Fix commands for `dns-txt` and `gen-peer-identity`

### PR DESCRIPTION
**Pull request by lemonlatte**
In the PR, it fixes the `dns-txt` command and update `gen-peer-identity` with the libp2p key.

### CODE REVIEW

File: command/bitmarkd/commands.go [hunk 1]
```diff
@@ -352,9 +368,19 @@ func dnsTXT(options *Configuration) {
...
+	if err != nil {
+		exitwithstatus.Message("error: cannot decode private key: %q  error: %s", peering.PrivateKey, err)
+	}
+
+	privateKey, err := crypto.UnmarshalPrivateKey(privateKeyBytes)
+	if err != nil {
+		exitwithstatus.Message("error: cannot generate private key: %q  error: %s", peering.PrivateKey, err)
+	}
+
+	peerID, err := peer.IDFromPublicKey(privateKey.GetPublic())
```
Comment (pieceofr) [inline command/bitmarkd/commands.go]: peer.IDFromPrivateKey(privateKey) may be more direct

File: util/peerKey.go [hunk 2]
```diff
@@ -7,15 +7,21 @@ import (
 	crypto "github.com/libp2p/go-libp2p-core/crypto"
 )
 
-//GenEd25519Key Generate a random ED29919, 2048 key
-func GenEd25519Key() (crypto.PrivKey, error) {
+//GenEd25519Key Generate a random ED29919
+func MakeEd25519PeerKey() (string, error) {
```
Comment (pieceofr) [inline util/peerKey.go]: In P2P module , there is a func GenRandPrvKey() , should we make them a util ?

Comment (lemonlatte) [reply util/peerKey.go]: The function `GenRandPrvKey` is redundant. We actually do not take any advantage of that wrap. When you use that function, you need to check error again. It basically a duplicated error checking.

File: command/bitmarkd/commands.go (hunk 1 above)
Comment (lemonlatte) [reply command/bitmarkd/commands.go]: Will fix.

File: util/peerKey.go (hunk 2 above)
Comment (pieceofr) [reply util/peerKey.go]: Agree. Another point is that should we move function identity.go in P2P into util/peerKey.go

Comment (lemonlatte) [reply util/peerKey.go]: Not sure. What is `identity.go` making for?

Comment (pieceofr) [reply util/peerKey.go]: Generate random key and decode/encode Hex Key from/to files

**Pull request merged**